src/app/home-component/home-component.component.html

```diff
@@ -0,0 +1,161 @@
+      <img class="tech-image" src="./assets/images/technology-logos/ansible-logo.png" alt="Salesforce">
+    </div>
+    <div class="custom-shape-divider-bottom-1702005791">
+      <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
+          <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
+      </svg>
+    </div>
+  </section>
+  
+  <!-- Contact Us Section -->
+  <section id="contact" class="contact-form">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6 contact-text">
+          <div>
+            <h2>Contact Us</h2>
+            <p class="contact-text-second-line">and we'll be in touch.</p>
+          </div>
+        </div>
+        <div class="col-md-6">
+          <div class="contact-form-wrapper">
+            <form action="https://api.web3forms.com/submit" method="POST" id="form">
+              <input type="hidden" name="access_key" value="d6023f15-d046-4301-bd42-c80bd9b99856" />
+              <input type="hidden" name="redirect" value="https://web3forms.com/success" />
+              <input type="checkbox" name="botcheck" id="" style="display: none;" />
+              <input type="email" name="email" class="form-control" placeholder="Your Email" required>
+              <input type="text" name="subject" class="form-control" placeholder="Subject" required>
+              <textarea name="message" class="form-control" rows="5" placeholder="Message" required></textarea>
+              <button type="submit" class="btn btn-primary">Submit</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="custom-shape-divider-bottom-1702005911">
+      <svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 120" preserveAspectRatio="none">
+          <path d="M985.66,92.83C906.67,72,823.78,31,743.84,14.19c-82.26-17.34-168.06-16.33-250.45.39-57.84,11.73-114,31.07-172,41.86A600.21,600.21,0,0,1,0,27.35V120H1200V95.8C1132.19,118.92,1055.71,111.31,985.66,92.83Z" class="shape-fill"></path>
+      </svg>
+    </div>
+  </section>
+  
+  <!-- Footer -->
+  <footer class="footer">
+    <div class="container text-center">
+      <img class="footer-logo" src="./assets/images/logo-vertical.png" alt="Segala Solutions Logo">
+      <p>gsegala@segalasolutions.com | 410-294-0545</p>
+    </div>
+  </footer>
+  
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+  
+  </body>
+  
+  <router-outlet></router-outlet>
```

Comment:
Router is what determines which page the subdirectory of the url goes to.

We want the router outlet to be in the app component.

The route localhost: http://localhost:4200/home should navigate to the home page component.

A navigation to  http://localhost:4200 should redirect to  http://localhost:4200/home

Reply:
Take a look at src/app/app-routing.module.ts. That is where the routes are configured. 

"Angular Router Outlet is a directive that acts as a placeholder in your Angular application where the content of routed components will be displayed. It dynamically renders the component associated with the current route, allowing for a single-page application (SPA) experience by updating the content based on the user's navigation without a full page reload."

Decent website explination: https://v17.angular.io/tutorial/tour-of-heroes/toh-pt5

There are probably good youtube videos too.

Reply:
i believe ive fixed it now